### PR TITLE
Feature/refworks rebrand

### PR DESCRIPTION
**Pull request by jvoytek**
https://trello.com/b/BWQgqU3o/flow

### CODE REVIEW

File: connectors/pme/pme_ui_dev.js [hunk 1]
```diff
@@ -244,12 +241,12 @@ var SaveToFlow = (function() {
...
-			doc.getElementById("stf_track_url").value = tracking.url;
-			doc.getElementById("stf_track_citation").value = tracking.citation;
-			doc.getElementById("stf_track_modified").value = JSON.stringify(tracking.modified);
-			//doc.getElementById("stf_tracking_form").submit();
+			doc.getElementById("s2r-track_found").value = tracking.found;
+			doc.getElementById("s2r-track_selected").value = tracking.selected;
+			doc.getElementById("s2r-track_url").value = tracking.url;
+			doc.getElementById("s2r-track_citation").value = tracking.citation;
+			doc.getElementById("s2r-track_modified").value = JSON.stringify(tracking.modified);
+			//doc.getElementById("s2r-tracking_form").submit();
```
Comment (aepheus) [inline connectors/pme/pme_ui_dev.js:249]: I get the feeling that this shouldn't be commented out (though I see it already was)

File: connectors/pme/pme_ui_dev.js [hunk 2]
```diff
@@ -462,20 +459,20 @@ var SaveToFlow = (function() {
...
-				countText = count ? count + " articles saved." : "1 article saved.";
+			var count = doc.getElementById("s2r-capture").getAttribute("data-saving-count"),
+				status = doc.getElementById("s2r-status"),
+				countText = count > 1 ? count + " articles saved to RefWorks" : "1 article saved to RefWorks";
 
-			doc.getElementById("stf_progress").style.display = "none";
+			doc.getElementById("s2r-progress").style.display = "none";
 			status.style.display = "block";
-			status.innerHTML = countText + '<form method="get" action="' + FLOW_SERVER + '/library/recent/" target="ProQuestFlow"><button id="stf_view_button" type="submit">View in Flow</button></form>';
+			status.innerHTML = s2r.Utils.template(s2rTemplateStrings.savedMessage)({'countText': countText, 'flowServer':FLOW_SERVER});
```
Comment (aepheus) [inline connectors/pme/pme_ui_dev.js:481]: I'm liking the template use.

**Comment by aepheus** [comment]
Doesn't look like we updated the icon that shows up in Firefox, that should be updated too (and the tooltip).

File: connectors/pme/pme_ui_dev.js (hunk 1 above)
Comment (jvoytek) [inline connectors/pme/pme_ui_dev.js:249]: Not knowing exactly how this is supposed to work I'm hesitant to just uncomment...Are we not getting tracking info from the Add On?

**Pull request closed without merging**